packages/client/wallets/aa/src/SmartWalletSDK.ts

```diff
@@ -18,17 +18,20 @@ export class SmartWalletSDK extends LoggerWrapper {
 
     private constructor(config: SmartWalletSDKInitParams) {
         super("SmartWalletSDK");
-        const validationResult = validateAPIKey(config.apiKey);
-        if (!validationResult.isValid) {
-            throw new Error("API key invalid");
-        }
-
-        this.crossmintService = new CrossmintWalletService(config.apiKey);
+        this.crossmintService = new CrossmintWalletService(config.clientApiKey);
         this.eaoWalletService = new EOAWalletService(this.crossmintService);
     }
 
-    static init(params: SmartWalletSDKInitParams): SmartWalletSDK {
-        return new SmartWalletSDK(params);
+    /**
+     * Initializes the SDK with the **client side** API key obtained from the Crossmint console.
```

Comment:
are we validating that it's an client side API key?

Reply:
No, but we should, will follow up w/ that later!